recipe/windows_cmake_install_bindir.patch

```diff
@@ -0,0 +1,13 @@
+diff --git CMakeLists.txt CMakeLists.txt
+index 3fd9d70..9ce3f67 100644
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -398,7 +398,7 @@ endif ()
+ # installation directories
+ if (OS_WINDOWS)
+   set (RUNTIME_INSTALL_DIR Bin)
+-  set (LIBRARY_INSTALL_DIR Lib)
++  set (LIBRARY_INSTALL_DIR Bin)
```

Comment:
Indeed. Could you please send an updated patch? I lack a Windows machine to test on. So this would be slow going for me.